docs/maintenance/newcomers.md

```diff
@@ -1,105 +1,34 @@
-# Newcomers Guide
+# New core developers and maintainers
 
 **Intended audience**
 
-The guide is intended as a crash course for new core developers of InvenioRDM with prior experience in running and installing InvenioRDM.
+The guide is intended as a crash course for new core developers/maintainers of InvenioRDM with prior experience in running and installing InvenioRDM.
 
-**Scope**
+## Scope
 
-The guide covers how we collaborate, the essential communication channels and key tools we use.
+Maintainers and core developers are trusted members of the project that regularly contribute to and manage the core codebase of InvenioRDM.
```

Comment:
Who maintainers are and what their duties and "powers" involve -anything from the 2025 Hamburg workshop- could fit well here.